packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -421,13 +416,22 @@ export async function uploadDataChunk<
       delay: 200,
       factor: 1.05,
       handleError(err, attemptContext) {
-        handleUploadDataChunkError({
-          err,
-          attemptContext,
-          logger,
-          batch,
-          uploadCorrelationId,
-        });
+        try {
```

Comment:
No functionality changes - just adding a warning log if our handleError function throws, which it seems to be doing rarely in prod.